common/logger/init.go

```diff
@@ -0,0 +1,94 @@
+package logger
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"os"
+	"path/filepath"
+)
+
+const (
+	DefaultTimeFormat           = "15:04:05.000"
+	DefaultCallerSkipFrameCount = 4
+
+	UseColor  = true
+	UseCaller = false // if you want to expose line of code of caller
+	LogFile   = ""    // if you want to store log to file
+)
+
+var (
+	logBuffer bytes.Buffer
+
+	// DebugMode flag for determining debug mode
+	DebugMode = false
+)
+
+func init() {
+	var (
+		lgr zerolog.Logger
+	)
+
+	zerolog.TimeFieldFormat = DefaultTimeFormat
+	zerolog.CallerSkipFrameCount = DefaultCallerSkipFrameCount
+
+	var writers zerolog.LevelWriter
+	consoleWriter := zerolog.ConsoleWriter{
+		Out:        &logBuffer,
+		NoColor:    UseColor,
```

Comment:
This field name and the const don't match.

Reply:
Updated, forget to add ! when using it : `NoColor : !UseColor`
But i've changed it to NoColor